### ISSUE TITLE
Help ME PLEASE!

### ISSUE THREAD

**Issue by grannysantiques** [Automattic/_s#1535]
About 4 years ago,  possibly longer, my former boss's son created a website for my businesses using your coding and basic template - www.michellesantiqueappraisals.com

It was great, looked amazing, and functioned. 

About 9 months ago the inserted text on the home page vanished and the "submit appraisal" form no longer works.

I have paid quite a bit of money to get it fixed and no one knows what to do. Finally, I determined that 1) Bootstrap is outdated 2) I noticed that y'all have updated some code and that needs to be done on my site.

It is built in WordPress and I have no idea where Boostrap is much less how to update it. I would like to do away with all of that and go with a stock template with some customization in color and function.

I have two businesses online and I just don't have the time to keep up with Bootstrap. I have tried to just apply a different template and it creates a "critical error." Basically, my website vanishes.

Any guidance that you can give me would be a great help. The slider at the top of the page needs to go but even deleting it makes the first page wonky.

Thank you so much for any assistance. The man who built the site doesn't respond to my calls for help and I need the $$ that the page used to generate.

Thank you,
Michelle

**Comment by rod3rix** [comment]
Hi Michelle, I can help you to fix and improve your website.
Please send me an email to help@zimple.pro
Rod

**Comment by grannysantiques** [comment]
Are you familiar with boostrap and this template?

**Comment by rod3rix** [comment]
Hi, I'm a front-end developer and I know very well Bootstrap. I'm working with the new version (5.x) and you have the 3.x but that is not a problem.
About the theme, it's a custom theme and I can edit it if is necessary; I created themes for my clients.
Please let me know and have a nice day

**Comment by grannysantiques** [comment]
The only page being affected is the home page everything else is working just fine. Could this be because bootstrap is so outdated?

**Comment by rod3rix** [comment]
No, the problem is different; to identify the issue it is necessary to review the admin of Wordpress.

**Comment by grannysantiques** [comment]
How much do you charge per hour?

**Comment by rod3rix** [comment]
$35

**Comment by grannysantiques** [comment]
Do you have an approximate time on how long it will take? We have had massive medical bills this year. Actually, I need to go to a Dr appt but I will let you know when I get back. I know that some of the plugins need to be deleted.

**Comment by rod3rix** [comment]
I hope you feel better.
To know how long will take, it's necessary to identify the issue and for that I need to check the Wordpress.

**Comment by nabilzidane** [comment]
Hi
In my opinion, the problem isn't related to the bootstrap version, there is so many websites still running under bootstrap v3.
I'm sure that I can help to identify the problem, so you can contact me at nabilzidane.info@gmail.com. or on fiverr if you want https://www.fiverr.com/s2/3911b915ed
Thanks

**Comment by grannysantiques** [comment]
Thank you so much for the response. I have been looking into some of the plugins on my site and flex slider, which I don't want anymore, might be part of the problem. It is on the home page only which happens to be the only page that isn't functioning. I am going to try to get rid of it and see if that helps if it doesn't I will contact you.
Cheers.

**Comment by nabilzidane** [comment]
Good idea, a lot of wordpress problems are caused by some plugins. I hope that will fix it. 
Good luck

**Comment by grannysantiques** [comment]
It's easy to get overloaded with plugins too. The site is very basic so I agree it is probably a plugin on the homepage. Time to clean house.

**Comment by okbutillsleep** [comment]
Hey @grannysantiques,

I'm not sure if this has been sorted but navigating to your site suggests to me it's not regarding the disappearing homepage text?

So just looking over your site I can see you're using the Gutenberg plugin, would I be correct in assuming you're using Gutenberg as your page editor?

When you edit the homepage, is the text there in the CMS? Assuming you're using Gutenberg, can you try adding a _Text_ block and typing something in the block and then updating the page to see if the changes are reflected on the page?

If this doesn't work it could point to your theme `themes/michelle-knows-antiques` template not rendering the text properly. 

First, to determine what template's are being loaded you could install the [What Template](https://wordpress.org/plugins/what-template/#description) plugin. This will tell you which template is being rendered when you navigate to your homepage.

From there, open this template in your editor and note the code. Make sure the template is set to output your CMS text, you could try using [get_the_content()](https://developer.wordpress.org/reference/functions/get_the_content/)

Hope this is a step forward but if this is advice you've already heard or incorrect then sorry to be nuisance!

**Comment by lucasmshepherd** [comment]
This thread is precious!

**Comment by grannysantiques** [comment]
Update, nothing has changed much other than the Rev Slider has been successfully deleted and I can't figure out how to use Gutenberg. I took a couple of weeks off to reboot my brain. @PantherMlem I will follow your suggestions and see what happens. When I was researching Bootstrap I found that a line of text needs to be in one of the gazillion footer files. I have looked through them all and there is nothing that looks like code. I will mess with it some more and report back. Thank you everyone.

**Comment by okbutillsleep** [comment]
Personally I doubt that Bootstrap will be effecting how your text outputs but let me know how you get on! :dizzy_face:
